### PR TITLE
Update ServiceContainer.php

### PR DESCRIPTION
**Pull request by buzz8year**
#216 
#217 
#222
#225  
#231 
#232 
#247 
#259 
#275

### CODE REVIEW

**Comment by lucacri** [comment]
+1

**Comment by lucacri** [comment]
Please merge this

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-    "name": "jonnyw/php-phantomjs",
-    "description": "A PHP wrapper for loading pages through PhantomJS",
+    "name": "cocostru/php-phantomjs",
```
Comment (BR0kEN-) [inline composer.json]: You should remove this and similar modifications in order to consider merge of this PR.

File: composer.json [hunk 2]
```diff
@@ -9,6 +9,11 @@
             "name": "Jonny Wenmoth",
             "email": "contact@jonnyw.me",
             "homepage": "http://jonnyw.me/"
+        },
+        {
+            "name": "Denis Kostukovic",
```
Comment (BR0kEN-) [inline composer.json]: Should be removed before merging to the main repository.

**Review by BR0kEN- — CHANGES_REQUESTED** [review]

**Comment by iantearle** [comment]
Please merge this. php-phantoms is broken without it.

**Comment by wcjonathan** [comment]
I've sent a tweet to the author to see if he'll merge this.

**Pull request merged**